src/rcdgen/check_data.cpp

```diff
@@ -1117,6 +1117,35 @@ static std::shared_ptr<TCORBlock> ConvertTCORNode(std::shared_ptr<NodeGroup> ng)
 }
 
 /**
+ * @tparam Type to cast the element to.
```

Comment:
Minor, but I think we'll want the parameter name, `T`, included for correct doxygen:
`@tparam T Type to cast the element to.`